_meta/beat.reference.yml

```diff
@@ -20,3 +20,6 @@ apm-server:
   #ssl.enabled: false
   #ssl.certificate : "path/to/cert"
   #ssl.key : "path/to/private_key"
+
+  # Rate limiter to limit number of events per second. 0 means disabled.
+  #ratelimit.size: 0
```

Comment:
Setting a default rate limit is difficult, but I think we should something large here so it will really only matter if something in the agent is broken or if there's a malicious usage. For example 10 per second seems like a high enough number that you'll never run into the rate limit under normal usage.